src/Components/SmartComponents/SystemsExposedTable/SystemsExposedTable.js

```diff
@@ -177,14 +177,27 @@ const SystemsExposedTable = (props) => {
         );
     };
 
+    const getSelectedSystemsWithName = () => {
+        if (!selectedHosts) {
```

Comment:
Also, since you returning in every statement it would be more clear to read it . 

if (check for one system) { return }
if (check for multiple systems) { return }

return [];